### PR TITLE
Save command done

### PR DESCRIPTION
**Pull request by Fyelight**
#68

### CODE REVIEW

File: src/main/java/seedu/doit/commons/exceptions/build.gradle [hunk 1]
```diff
@@ -0,0 +1,187 @@
+/*
+ * Gradle Configuration File
```
Comment (jinshunlee) [inline src/main/java/seedu/doit/commons/exceptions/build.gradle]: Why is this here? @Fyelight

File: src/main/java/seedu/doit/logic/parser/Parser.java [hunk 2]
```diff
@@ -32,7 +33,8 @@
     /**
      * Parses user input into command for execution.
      *
-     * @param userInput full user input string
+     * @param userInput
+     *            full user input string
```
Comment (jinshunlee) [inline src/main/java/seedu/doit/logic/parser/Parser.java:37]: Can put them together? why split them up

**Review by jinshunlee — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/doit/logic/parser/Parser.java (hunk 2 above)
Comment (Fyelight) [reply src/main/java/seedu/doit/logic/parser/Parser.java:37]: This is by auto-indentation

**Review by jinshunlee — APPROVED** [review]

**Review by jinshunlee — APPROVED** [review]
